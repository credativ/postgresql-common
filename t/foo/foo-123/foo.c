#include "postgres.h"
#include "fmgr.h"
#include "utils/builtins.h"

PG_MODULE_MAGIC;

PG_FUNCTION_INFO_V1 (foo);

Datum
foo (PG_FUNCTION_ARGS)
{
    PG_RETURN_TEXT_P(cstring_to_text("bar"));
}
