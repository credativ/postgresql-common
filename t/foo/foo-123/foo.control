default_version = '123'
comment = 'The foo extension'
