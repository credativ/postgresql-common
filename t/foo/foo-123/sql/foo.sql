CREATE EXTENSION foo;

SELECT foo();

DROP EXTENSION foo;
