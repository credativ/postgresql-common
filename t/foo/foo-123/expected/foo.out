CREATE EXTENSION foo;
SELECT foo();
 foo 
-----
 bar
(1 row)

DROP EXTENSION foo;
